docker/mwdb.ini

```
@@ -6,3 +6,4 @@ uploads_folder = /app/uploads
 serve_web = 0
 local_plugins_folder = /app/plugins
 local_plugins_autodiscover = 1
+statement_timeout = 4000
```

May be a bit longer? 15 seconds sounds reasonable.
```suggestion
statement_timeout = 15000
```

---

Ok - changed. Previously statement_timeout was 4000 because of tests reason - it had to be less than 8000 (hardcoded request timeout).